vuln/npm/390.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": "390",
+  "title": "Cross-site Scripting (XSS) - Stored",
+  "overview": "[simple-server] HTML with iframe element can be used as filename, which might lead to load and execute malicious JavaScript ",
+  "created_at": "2018-01-26",
```

Comment:
The dates formats are inconsistent across different JSON files. Some of them like https://github.com/nodejs/security-wg/blob/master/vuln/npm/102.json#L3 have date and time and some others like this one just plain dates. Not a big deal.

Reply:
The original NSP dump is in ISO8601 format while the other reports that we've started issuing are in a simpler one. Indeed a consistency which I think we can further discuss on the other issues we've got open with regards to machine readable and automation to help us with this process.

Reply:
Related to https://github.com/nodejs/security-wg/issues/115.